pricing-page-exercise/onur/index.html

```diff
@@ -4,10 +4,120 @@
   <title>Pricing page Exercise</title>
 </head>
 <body>
-  
+  <header>
+    <nav class="navbar">
+        <div><h3 class="heading-small">Company name</h3></div>
+        <ul>  
+            <li><a class="navlink" href="#Features">Features</a></li>
+            <li><a class="navlink" href="#Enterprise">Enterprise</a></li>
+            <li><a class="navlink" href="#Support">Support</a></li>
+            <li><a class="navlink" href="#Pricing">Pricing</a></li>
+            <li><a class="btn" href="#">Sign Up</a></li>
+        </ul>  
+    </nav>
+ </header>
+ <main>
+    <div class="description">
+        <h1 id="pricing" class="text-center">Pricing</h1>
+        <p class="text-center">Quickly build an effective pricing table for your potential customers with this Bootstrap example. It's built with default Bootstrap components and utilities with little customization.</p>
+    </div>
+    <div class="total-content">
+        <div class="content">
+            <div class="prices">
+                <div class="card_head">
+                    <h1 class="text-center">Free</h1>
+                </div>
+                <div class="card_body">
+                    <h1><span class="dollar">&#36;0</span><span>/ mo</span></h1>
+                    <ul>
+                        <li>10 users included</li>
+                        <li>2 GB of storage</li>
+                        <li>Email support</li>
+                        <li>Help center access</li>
+                    </ul>
+                    <button class="btn price-btn">Sign up for free</button>
+                </div>
+            </div>
+            <div class="prices">
+                <div class="card_head">
+                    <h1 class="text-center">Pro</h1>
+                </div>
+                <div class="card_body">
+                    <h1><span class="dollar">&#36;15 </span><span>/ mo</span></h1>
+                    <ul>
+                        <li>20 users included</li>
+                        <li>10 GB of storage</li>
+                        <li>Priority email support</li>
+                        <li>Help center access</li>
+                    </ul>
+                    <button class="btn price-btn blue-btn">Get started</button>
+                </div>
+            </div>
+            <div class="prices">
+                <div class="card_head">
+                    <h1 class="text-center">Enterprise</h1>
+                </div>
+                <div class="card_body">
+                    <h1><span class="dollar">&#36;29</span><span>/ mo</span></h1>
+                    <ul>
+                        <li>30 users included</li>
+                        <li>15 GB of storage</li>
+                        <li>Phone and email support</li>
+                        <li>Help center access</li>
+                    </ul>
+                    <button class="btn price-btn blue-btn">Contact us</button>
+                </div>
+            </div>
+        </div>
+        <footer>
```

Comment:
There's a problem with opening/closing tags. I think you placed footer element too early (before closing main element).